RecoLocalTracker/SiStripClusterizer/python/SiStripClusters2ApproxClusters_cff.py

```diff
@@ -1,5 +1,5 @@
 from RecoLocalTracker.SiStripClusterizer.SiStripClusters2ApproxClusters_cfi import *
 
 from Configuration.ProcessModifiers.approxSiStripClusters_cff import approxSiStripClusters
-SiStripClusters2ApproxClustersHLT = SiStripClusters2ApproxClusters.clone()
-approxSiStripClusters.toModify(SiStripClusters2ApproxClustersHLT, inputClusters = "siStripClustersHLT")
+hltSiStripClusters2ApproxClusters = SiStripClusters2ApproxClusters.clone()
+approxSiStripClusters.toModify(hltSiStripClusters2ApproxClusters, inputClusters = "siStripClustersHLT")
```

Comment:
Why is this needed?

The convention (the way I know it) is that modules inside the HLT menu are normally called `hlt*`, while the opposite applies to modules in other steps of the chain, like this one. (cc: @Martin-Grunewald)

Reply:
This is the name of the collection that will be produced at the HLT.  You can see the JIRA ticket linked in the PR description for details.  In this particular workflow we are just creating the collection offline for testing purposes.

Reply:
@missirol @FHead Perhaps you can advise whether we stick with this naming convention or whether the name will change on the HLT side.

Reply:
Module labels at HLT start with ```hlt```. I believe we have settled on ```hltSiStripClusters2ApproxClusters``` for that module.

Reply:
Ok, so I keep this PR as it is. Thanks.